venue/models.py

```diff
@@ -244,6 +244,25 @@ class Ranking(models.Model):
     class Meta:
         get_latest_by = 'timestamp'
 
+    @classmethod
+    def clean_ranking(cls):
+        """
+        Remove lines older than 7 days and duplicates for each day
```

Comment:
@alexdashkov I've thought about how much data I'm sure that we are now removing, and don't want to remove entries older than 7 days. Disk space is cheap enough that storing 1 entry per user per day is not a problem. As long as the data storage increases in terms of days and not minutes, we are good.